ci/authn-azure/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+AWS_PEM_FILE_LOCATION="<location of pem file to login to machine>"
+USERNAME="<username of AWS machine>"
+AWS_MACHINE="<AWS machine>"
+
+pushd ../..
+  . version_utils.sh
+  CONJUR_IMAGE_VERSION="$(version_tag)"
```

Comment:
I would make this one statement `  export CONJUR_IMAGE_VERSION="$(version_tag)"` like you did this in the previous file on line 23 (`export CONJUR_DATA_KEY="$(< data_key)"`)